atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/NoDuplicateWritesTransaction.java

```diff
@@ -50,7 +50,7 @@
             new CacheLoader<TableReference, Map<Cell, byte[]>>() {
                 @Override
                 public Map<Cell, byte[]> load(TableReference input) {
-                    return Collections.synchronizedMap(Maps.<Cell, byte[]>newHashMap());
+                    return Collections.synchronizedMap(Maps.newHashMap());
```

Comment:
Extranneous diff?

Reply:
Not really a problem, just a small cleanup.